base/strings/search.jl

```diff
@@ -44,7 +44,7 @@ function search(s::AbstractString, c::Chars, i::Integer)
 end
 search(s::AbstractString, c::Chars) = search(s,c,start(s))
 
-in(c::Char, s::AbstractString) = (search(s,c)!=0)
+in(c::Char, s::AbstractString) = !isequal(search(s,c), 0)
```

Comment:
This should also not change, since it isn't comparing `c`; it's testing the result of a different function.